src/sagemaker/clarify.py

```diff
@@ -1399,6 +1580,45 @@ def get_explainability_config(self):
         return copy.deepcopy({"shap": self.shap_config})
 
 
+class AsymmetricSHAPConfig(ExplainabilityConfig):
+    """Config class for Asymmetric SHAP algorithm for TimeSeries explainability"""
+
+    def __init__(
+        self,
+        explanation_type: str = ASYM_SHAP_DEFAULT_EXPLANATION_TYPE,
+        num_samples: Optional[int] = ASYM_SHAP_DEFAULT_NUM_SAMPLES,
+    ):
+        """Initialises config for asymmetric SHAP config.
+
+        AsymmetricSHAPConfig is used specifically and only for TimeSeries explainability purposes.
+
+        Args:
+            explanation_type (str): Type of explanation to be used
```

Comment:
People refer to docs to understand how to use tools. Please, list the possible values

Reply:
Will address today.